CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 1.2.0
+
+## Features and improvements
+* Add `*-sort-key` layout property for circle, fill, and line layers, to sort the z-index of features within a single layer([#8467](https://github.com/mapbox/mapbox-gl-js/pull/8467))
```

Comment:
I'm not sure how much this distinction matters, but `*-sort-key` impacts the draw order of features which is what makes it appear as though it's possible to sort features within a layer.  Now that I'm reading this changelog entry with the addition of "z-index," it adds a level of precision that now makes the changelog entry feel less accurate to me, but I would defer to @ahk on this.

I'd probably prefer "...to control the draw order of features..." or the more general "to sort features within a single layer."

Reply:
How about "Add `*-sort-key` layout property for circle, fill, and line layers. Allows specifying the order that features are drawn within a layer."